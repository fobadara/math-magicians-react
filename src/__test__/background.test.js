import { render, cleanup } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Background from '../components/background';

afterEach(cleanup);

describe('Background', () => {
  test('should create snapshot', () => {
    const { asFragment } = render(<Router><Background /></Router>);
    expect(asFragment(<Background />)).toMatchSnapshot();
  });
});
