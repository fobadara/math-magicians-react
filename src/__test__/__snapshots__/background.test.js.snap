// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background should create snapshot 1`] = `
<DocumentFragment>
  <div
    class="background"
  >
    <div>
      <div
        class="cube"
      />
      <div
        class="cube"
      />
      <div
        class="cube"
      />
      <div
        class="cube"
      />
      <div
        class="cube"
      />
    </div>
    <header>
      <div
        class="nav-border"
      >
        <nav
          class="navbar navbar-expand-lg navbar-light"
        >
          <div
            class="container"
          >
            <a
              class="navbar-brand"
              href="/"
            >
              Math Magicians
            </a>
            <button
              aria-controls="responsive-navbar-nav"
              aria-label="Toggle navigation"
              class="navbar-toggler collapsed"
              type="button"
            >
              <span
                class="navbar-toggler-icon"
              />
            </button>
            <div
              class="navbar-collapse collapse"
              id="responsive-navbar-nav"
            >
              <div
                class="me-auto navbar-nav"
              />
              <div
                class="navbar-nav"
              >
                <a
                  class="active nav-link"
                  data-rr-ui-event-key="/"
                  data-testid="home"
                  href="/"
                  id="/"
                >
                  Home
                </a>
                <a
                  class="nav-link"
                  data-rr-ui-event-key="calculator"
                  data-testid="quotes"
                  href="/calculator"
                  id="calculator"
                >
                  Calculator
                </a>
                <a
                  class="nav-link"
                  data-rr-ui-event-key="quotes"
                  data-testid="quotes"
                  href="/quotes"
                  id="quotes"
                >
                  Quotes
                </a>
              </div>
            </div>
          </div>
        </nav>
      </div>
    </header>
    <main
      class="container"
    >
      <div
        class=""
      >
        <h1>
          Welcome to our page!
        </h1>
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat dolor ratione nulla expedita eligendi error quo corrupti. Id cum placeat voluptate totam modi quaerat eveniet voluptatum ab fuga ullam, aliquam aliquid officia ad in enim earum aut saepe non sit maiores? Officia in explicabo illo similique natus ea voluptatibus eveniet.
        </p>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Enim, praesentium. Impedit eveniet tenetur illum fugiat harum hic molestias ipsum explicabo exercitationem aspernatur officiis eligendi rerum tempora excepturi nisi debitis voluptate, temporibus, in magnam voluptatem autem? Facilis, dolor? Provident, tenetur labore. Veniam nobis, inventore libero quis tenetur est enim iste nisi!
        </p>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
